azDeploySecureSub.bicep

```diff
@@ -35,6 +35,8 @@ var webAppName = nameModule[2].outputs.shortName
 var kvName = nameModule[3].outputs.shortName
 var sqlName = nameModule[4].outputs.shortName
 var planName = nameModule[5].outputs.shortName
+var lawName = nameModule[6].outputs.shortName
```

Comment:
Just a note that I added additional names too (for the deployment script and the UAMI), which will result in merge conflicts. We'll have to pay attention to the index when resolving them.